plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -288,6 +300,44 @@ static void ffmpeg_source_open(struct ffmpeg_source *s)
 	}
 }
 
+static void ffmpeg_source_start(struct ffmpeg_source *s)
+{
+	if (!s->media_valid)
+		ffmpeg_source_open(s);
+
+	if (!s->media_valid)
+		return;
+
+	mp_media_play(&s->media, s->is_looping);
+	if (s->is_local_file)
```

Comment:
Request from @RytoEX:

```suggestion

	if (s->is_local_file)
```

Reply:
I actually didn't request this.  I requested a blank line _below_.